src/classlibnative/bcltype/system.cpp

```diff
@@ -73,9 +73,11 @@ FCIMPLEND;
 
 
 
-#ifndef FEATURE_CORECLR
 INT64 QCALLTYPE SystemNative::GetWorkingSet()
 {
+#if PLATFORM_UNIX
+    return 0; // TODO: Implement in PAL
```

Comment:
I also think it is worth creating an public API in the implementation that isn't exposed in contracts that exposes just the minimum we need from the runtime and implement the rest of this in corefx. 

I also agree that we need to be careful about requiring reflection for too many APIs in this library.